docs/tutorials/HPCIntro.md

```diff
@@ -289,4 +289,3 @@ You might find the following tutorials interesting to tackle next:
 
 * [Persisting Data Between Restarts](PersistentServer)
 * [Server Mode](ServerMode.md)
-* If you'd like to configure Cromwell to use a local scratch device, see instructions here [HPCSlurmWithLocalScratch.md](HPCSlurmWithLocalScratch.md)
```

Comment:
Why is this obsolete? This was added a couple years ago by our Intel friends: https://github.com/broadinstitute/cromwell/pull/6702

Does this not work anymore?

Reply:
I removed it because it includes editing the source of `RuntimeEnvironment.scala`, which is now deleted.

If we have contact info with the author, we can certainly figure something out; it was a judgment call on my part whether to leave an up inoperable tutorial or take it down until it's fixed.

Reply:
I do have contact info for her, I'll shoot her an email and see if she cares to update these instructions.